packages/web/src/common/store/cache/collections/createPlaylistSaga.ts

```diff
@@ -19,9 +19,10 @@ import {
   newCollectionMetadata,
   confirmerActions,
   confirmTransaction,
-  RequestConfirmationError
+  RequestConfirmationError,
+  put
```

Comment:
I worry this will get confusing and people will forget. Do we only need it when doing cache adds? What happens when we forget?

Reply:
yeah it's super confusing. basically folks should literally not need to use it, it was just that redux-saga types don't know how to deal with dispatch allowing a ThunkAction. so i think will only be needed for the few times we call `put(add(...))`

Reply:
hmm ok I guess that's fine. Wondering if there's a place we can put a comment so someone dealing with the type error won't waste time on it

Reply:
added a comment :)